core/chains/evm/txmgr/finalizer.go

```diff
@@ -23,6 +24,7 @@ var _ Finalizer = (*evmFinalizer)(nil)
 const processHeadTimeout = 10 * time.Minute
 
 type finalizerTxStore interface {
+	// FindConfirmedTxesReceipts fetch the txes with partially loaded receipts for performance reason
```

Comment:
This method actually only fetches the receipts
```suggestion
	// FindConfirmedTxesReceipts fetches partially loaded receipts for performance reason
```